### ISSUE TITLE
Evaluate adding jamal to the project

### ISSUE THREAD

**Issue by Michael1993** [junit-pioneer/junit-pioneer#756]
[Jamal](https://github.com/verhas/jamal) is a meta markup language that you can use to enhance some textfiles like Markdown, XML or AsciiDoc.

We could use it for example to get the version information automatically.

**Comment by Bukama** [comment]
So a markup language for a markup language. Hmm

**Comment by Michael1993** [comment]
I'm not sure what else to call it - it's a macro language that works purely on text.

**Comment by Bukama** [comment]
> I'm not sure what else to call it - it's a macro language that works purely on text.

Yeah, but I'm not sure if we add another markup language on top of AsciiDoc (given the fact, that we have an issue to change the markdown to AsciiDoc).

**Comment by beatngu13** [comment]
JUnit Jupiter uses Gradle/Asciidoctor for this, e.g. to insert the current version:

https://github.com/junit-team/junit5/blob/7082d3d9169cf06c559bd86f723f95c9a597a704/documentation/documentation.gradle.kts#L231

Example uses `bom-version`:

https://github.com/junit-team/junit5/blob/7082d3d9169cf06c559bd86f723f95c9a597a704/documentation/src/docs/asciidoc/user-guide/running-tests.adoc?plain=1#L31

**Comment by Michael1993** [comment]
> Yeah, but I'm not sure if we add another markup language on top of AsciiDoc (given the fact, that we have an issue to change the markdown to AsciiDoc).

Jamal is non-invasive, meaning if we don't like/want it, we can just delete the `.jam` files.
